src/main/java/seedu/address/model/person/person/UniquePersonList.java

```diff
@@ -104,6 +104,11 @@ public void setPersons(List<Person> persons) {
         sortList();
     }
 
+    /** Returns the internal list of contacts */
+    public ObservableList<Person> getInternalContactList() {
```

Comment:
I understand its wrong but, would it be better for the method to be called getInternalPersonList() instead? since the other methods are setPerson and file is called UniquePersonList.